src/test/java/org/jpeek/MetricsTest.java

```diff
@@ -160,17 +160,17 @@
             new Object[] {"TwoCommonAttributes", "TLCOM", 4.0d},
             new Object[] {"WithoutAttributes", "TLCOM", 1.0d},
             new Object[] {"MethodMethodCalls", "LCOM4", 0.6d},
-            new Object[] {"Bar", "LCC", 0.0d},
-            new Object[] {"Foo", "LCC", 1.0d},
-            new Object[] {"MethodMethodCalls", "LCC", 0.1d},
-            new Object[] {"MethodsWithDiffParamTypes", "LCC", 0.2d},
-            new Object[] {"NoMethods", "LCC", 0.0d},
-            new Object[] {"OneMethodCreatesLambda", "LCC", 0.0d},
-            new Object[] {"OneVoidMethodWithoutParams", "LCC", 0.0d},
-            new Object[] {"OnlyOneMethodWithParams", "LCC", 0.0d},
-            new Object[] {"OverloadMethods", "LCC", 1.0d},
-            new Object[] {"TwoCommonAttributes", "LCC", 0.0d},
-            new Object[] {"WithoutAttributes", "LCC", 0.0d}
+            new Object[] {"Bar", "MWE", 1.0d},
```

Comment:
@uurl Why removing LCC metrics?

Reply:
@paulodamaso By two reasons:
1. When I did this PR, the LCC metrics test was not implemented yet.
2. There is a limit in the method size, if it is longer than 100 LOC, TravisCI marks an error, so I opened this issue #233
I we just add more lines to that method it violates TravisCI code rules.

Reply:
@uurl Right, you should do this way then: merge your changes keeping LCC metrics and put a comment for static analisys tools ignore LOC count check on this class.

Reply:
@paulodamaso sorry, I don't know how to do that

Reply:
@uurl The tags for ignoring some specific validations are: `@checkstyle` and  `@SupressWarnings("PMD.SomeCheck")` ; there's some examples in jpeek code database.